server/core/region_tree_test.go

```diff
@@ -67,16 +67,16 @@ func (s *testRegionSuite) TestRegionInfo(c *C) {
 		Id:      n,
 		StoreId: n,
 	}
-	r.Peers = append(r.Peers, removePeer)
+	r.meta.Peers = append(r.meta.Peers, removePeer)
```

Comment:
when do we use `meta` or`GetMeta()`?

Reply:
the test is in the `core` package, I do not change its behavior in the test of this package.